pacsman/integration_test.py

```diff
@@ -47,6 +49,7 @@ def initialize_filesystem_client(dicom_dir, *args, **kwargs):
 LOCAL_PACS_URL = os.environ.get('LOCAL_PACS_URL', 'localhost')
 
 
+
```

Comment:
`flake8` causes the build to fail because of this (https://travis-ci.org/innolitics/pacsman/builds/437162035?utm_source=github_status&utm_medium=notification), along with a couple other formatting issues.